### ISSUE TITLE
[bug]: Docker install errors "Space_BASE_URL" variable is not set

### ISSUE THREAD

**Issue by sovereignalmida** [makeplane/plane#4813]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

mkdir plane-selfhost

cd plane-selfhost

curl -fsSL -o setup.sh https://raw.githubusercontent.com/makeplane/plane/master/deploy/selfhost/install.sh

chmod +x setup.sh

at both option 1 and option 2 after I get the following errors.  Step 1 did finally complete - still waiting on Step 2 (Start) 

WARN[0003] The "SPACE_BASE_URL" variable is not set. Defaulting to a blank string. 
WARN[0003] The "APP_BASE_URL" variable is not set. Defaulting to a blank string. 

### Steps to reproduce

mkdir plane-selfhost

cd plane-selfhost

curl -fsSL -o setup.sh https://raw.githubusercontent.com/makeplane/plane/master/deploy/selfhost/install.sh

chmod +x setup.sh
 Select Step 1
 
 see error  OR
 do above select step 2 and see error

### Environment

Production

### Browser

Other

### Variant

Self-hosted

### Version

v0.21-dev

**Comment by akshat5302** [comment]
Closing this as stale: there hasn't been any activity here for a while, and it was reported against a release that is well behind current Plane (Community v1.4.2 / Commercial v3.2.0).

If you still hit this on a current version, please open a new issue with your Plane version, install method and container logs, and link back here. Thanks!
